arch/arm/soc/st_stm32/stm32f0/soc.h

```diff
@@ -48,6 +48,19 @@
 #define CONFIG_PINCTRL_STM32_PREFIX ST_STM32_PINCTRL_48000000
 #endif
 
+#ifdef CONFIG_GPIO_PINCTRL_STM32
+#include <stm32f0xx_ll_bus.h>
+#include <stm32f0xx_ll_exti.h>
+#include <stm32f0xx_ll_gpio.h>
+#include <stm32f0xx_ll_system.h>
+#define CONFIG_GPIO_STM32_GPIOA_PREFIX ST_STM32_GPIO_PINCTRL_48000000
```

Comment:
Same as for pinctrl, can it be derived from dts?

Reply:
Yes it can. Will change.